cmds/netcat/netcat_test.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"testing"
+	"time"
+)
+
+var Host = "127.0.0.1"
```

Comment:
For some reason, the test doesn't works fine for IPV6 loopback. I'll try improve that.